syne_tune/optimizer/schedulers/searchers/bayesopt/gpautograd/gp_regression.py

```diff
@@ -27,9 +27,11 @@
     GaussianProcessMarginalLikelihood,
 )
 from syne_tune.optimizer.schedulers.searchers.bayesopt.gpautograd.mean import (
-    ScalarMeanFunction,
     MeanFunction,
 )
+from syne_tune.optimizer.schedulers.searchers.bayesopt.gpautograd.target_transform import (
+    ScalarTargetTransform,
```

Comment:
nit: remove comma and black will beautify correctly (unless the import is indeed that long)